ee/localserver/krypto-ec-middleware.go

```diff
@@ -303,7 +304,8 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		// it's possible the keys will be noop keys, then they will error or give nil when crypto.Signer funcs are called
 		// krypto library has a nil check for the object but not the funcs, so if are getting nil from the funcs, just
 		// pass nil to krypto
-		if e.hardwareSigner != nil && e.hardwareSigner.Public() != nil {
+		// hardware signing is not implemented for darwin
+		if runtime.GOOS != "darwin" && e.hardwareSigner != nil && e.hardwareSigner.Public() != nil {
```

Comment:
Why change this?

Reply:
In the current state, the signer will not be nil, but will error when sign function is called. So it didn't make sense to me to just let it error.